zebra/zebra_rib.c

```diff
@@ -1424,22 +1424,20 @@ static void rib_process_update_fib(struct zebra_vrf *zvrf,
 		if (!nh_active) {
 			if (IS_ZEBRA_DEBUG_RIB) {
 				char buf[SRCDEST2STR_BUFFER];
+				const char *nhi = "nexthop inactive";
```

Comment:
Wouldn't it just be better to have this as part of the error string instead of making it a %s parameter?